lib/gpprmsya.gi

```diff
@@ -673,14 +673,8 @@ InstallMethod(IsNaturalAlternatingGroup,"knows size",true,[IsPermGroup
 end );
 
 InstallMethod(IsNaturalAlternatingGroup,"comprehensive",true,[IsPermGroup],0,
-function( grp )
-  if 0 = NrMovedPoints(grp)  then
-    return IsTrivial(grp);
-  else
-    return PermgpContainsAn(grp)=true and
-      ForAll(GeneratorsOfGroup(grp),i->SignPerm(i)=1);
-  fi;
-end );
+G -> IsTrivial(G) or
+  ForAll(GeneratorsOfGroup(G),i->SignPerm(i)=1) and PermgpContainsAn(G)=true);
```

Comment:
Perhaps also like this?
```suggestion
  ForAll(GeneratorsOfGroup(G),i->SignPerm(i)=1) and PermgpContainsAn(G));
```

Reply:
`PermgpContainsAn` can return either `true` or `false` or the return value of `PermNatAnTestDetect` which can be a set of points, so I can't make that change (as much as I'd like to).